network/simulation/node.go

```diff
@@ -83,6 +83,13 @@ func AddNodeWithMsgEvents(enable bool) AddNodeOption {
 	}
 }
 
+// AddNodeAsBootNode toggles whether the node will be configured as a bootnode
+func AddNodeAsBootNode(enable bool) AddNodeOption {
```

Comment:
mega nitpick but sometimes you write `BootNode` and sometimes `Bootnode`.
I think in other places in the codebase it was always `Bootnode` so let's keep to that